tests/Agent/IntegrationTests/Applications/RejitMvcApplication/Properties/AssemblyInfo.cs

```diff
@@ -13,7 +12,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("RejitMvcApplication")]
-[assembly: AssemblyCopyright("Copyright ©  2020")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```

Comment:
Was this intentional?

Reply:
that's when it was originally created. we have other apps with that too.